### ISSUE TITLE
Recipient of a message that is sent right before sender closes the trade as complete, should be notified and be able to read it

### ISSUE THREAD

**Issue by slrslr** [haveno-dex/haveno#1450]
Trade process is good also the messaging between the traders, but there is one uncertainty/confusion.

When I receive payment of other party, it happens that i want to send a message to that person, so I send it and then close the trade as complete so it does not confuse/distract me on the open trades tab.

Even if I do that, I think that the recipient should be notified about my message, even if he was away from keyboard (AFK) at the time message was sent/trade was completed and If the recipient reply to my message, i would like to also be notified about that reply. Maybe communication can go on for limited time or limited amount of messages.

**Comment by charcharchar** [comment]
Yeah. I always found this a bit annoying as its so easy to hit close and not even realize you are blocking potential messages from your peer immediately after you have just completed the trade, very sudden and may miss out on any final pleasantries/end trade chat. Maybe a warning to say this will mean you cant chat with your peer when you hit close and give a grace time of 1 hour after you hit close? And then if someone cant wait an hour, have another button to completely finalize trade now.

**Comment by charcharchar** [comment]
some trades are not interactive so one hour is probably way to early. maybe its not needed at all as long as there is warning that closing will stop all comms.
